src/field_theory/galois.lean

```diff
@@ -143,6 +146,9 @@ lemma alg_equiv.transfer_galois (f : E ≃ₐ[F] E') : is_galois F E ↔ is_galo
 lemma is_galois_iff_is_galois_top : is_galois F (⊤ : intermediate_field F E) ↔ is_galois F E :=
 (intermediate_field.top_equiv).transfer_galois
 
+instance is_galois_bot : is_galois F (⊥ : intermediate_field F E) :=
```

Comment:
Do you want analogies of this for normal and separable as well?

Reply:
I guess the reason why it didn't occur to me is that the separable and normal files don't have anything involving the intermediate_field type. I could add analogs though.

Reply:
Hmm... I would expect that sooner or later we'll want to talk about separable intermediate extensions... but maybe not yet.
Anyway, I guess it would be better to do a separate PR for such an API (instead of adding just little tidbits on random occasions).
So I'll merge this.